two-factor.php

```diff
@@ -27,6 +27,24 @@ function wpcom_vip_is_two_factor_forced() {
 		return false;
 	}
 
+	// Don't force 2FA for OneLogin SSO
+	if ( function_exists( 'is_saml_enabled' ) && is_saml_enabled() ) {
+		return false;
+	}
+
+	// Don't force 2FA for SimpleSaml
+	if ( function_exists( '\HumanMade\SimpleSaml\instance()' ) ) {
```

Comment:
For this, is it simply enough that that method exists? Or do we need to actually check if it's enabled?

Reply:
Good point. It looks like we need to bail if the instance function return falsey, which indicates there's not a valid SAML config.

https://github.com/wpcomvip/skyscanner/blob/940e0be2dcf2fc9891db4fbe0ae96057d827cc6a/client-mu-plugins/wp-simple-saml/inc/namespace.php#L156-L159